serving/docker/scripts/install_inferentia2.sh

```diff
@@ -0,0 +1,47 @@
+apt-get update -y
+apt-get install -y wget \
+                   curl \
+                   gnupg2 \
+                   python3-pip \
+                   git
+
+# Configure Linux for Neuron repository updates
+. /etc/os-release
+tee /etc/apt/sources.list.d/neuron.list > /dev/null <<EOF
+deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main
+EOF
+wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | apt-key add -
+
+apt-get update -y && apt-get install aws-neuronx-dkms=2.* \
+                                     aws-neuronx-collectives=2.* \
+                                     aws-neuronx-runtime-lib=2.* \
+                                     aws-neuronx-tools=2.* -y
+
+export PATH=/opt/aws/neuron/bin:$PATH
+
+# Install pip
+ln -sf /usr/bin/python3.8 /usr/bin/python
```

Comment:
python and pip installation can reuse DJLServing's install_python script

Reply:
Done